src/braket/ocean_plugin/braket_dwave_sampler.py

```diff
@@ -134,15 +134,30 @@ def sample_ising(
 
         Examples:
             This example submits a two-variable Ising problem mapped directly to qubits
-            0 and 1.
+            0 and 1 on a D-Wave 2000Q device.
 
-            >>> from braket.ocean_plugin import BraketDWaveSampler
-            >>> sampler = BraketDWaveSampler(s3_destination_folder)
-            >>> sampleset = sampler.sample_ising({0: -1, 1: 1}, {}, answer_mode="HISTOGRAM")
+            >>> from braket.ocean_plugin import BraketSampler
+            >>> device_arn_1 = "arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6"
+            >>> sampler = BraketSampler(s3_destination_folder, device_arn_1)
+            >>> H = {0: -1, 1: 1}
+            >>> sampleset = sampler.sample_ising(H, {}, resultFormat="HISTOGRAM")
```

Comment:
The parameter should be named `h` instead of `H`. The keyword arguments in these examples should remain unchanged to match D-Wave behavior. i.e. keep `num_reads` over `shots`.